### DISCUSSION TITLE
Can spec generation for class-based-views be disabled?

### DISCUSSION THREAD

**Discussion by rod7760** [apiflask/apiflask#378] [Q&A] [answered]
Hello @greyli 

I see in 2021 you talked about class-based-views [here](https://github.com/apiflask/apiflask/discussions/14#discussioncomment-572506).

What is the current state of this functionality? Can I use `@app.doc(hide=True)` with a class based view? 

edit:

The docs do talk about `MethodView` [here](https://apiflask.com/usage/?query=class+based+view#use-class-based-views), but is there support for `View`?

**Comment by greyli** [comment, marked as answer]
`@app.doc(hide=True)` should work with class-based views.

`View` is not supported, it seems impossible to fetch method information (GET/POST/...) from `View` classes.

**Reply by rod7760** [reply]
Hi @greyli. that makes sense. I should have closed this after making https://github.com/apiflask/apiflask/issues/379.
